config/src/test/java/org/springframework/security/config/annotation/web/reactive/EnableWebFluxSecurityTests.java

```diff
@@ -153,6 +153,7 @@ public void defaultPopulatesReactorContext() {
 		client
 			.get()
 			.uri("/")
+			.headers(headers -> headers.setBasicAuth("user", "password"))
```

Comment:
Why was this added? It seems like an odd side-effect of the change.

Reply:
yes, you are right